store/tikv/lock_resolver.go

```diff
@@ -206,7 +207,8 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 			continue
 		}
 
-		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary, 0)
+		// Use math.MaxUint64 as currentTS, so the txn is either committed or rollbacked.
+		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary, 0, math.MaxUint64)
```

Comment:
Why not just use 0 current TS?

Reply:
In `kvproto.Cleanup`, current ts = 0 will rollback the transaction (for compatibility reason), while `kvproto.CheckTxnStatus` doesn't has this implication.
Now we use `kvproto.CheckTxnStatus` to implement the `getTxnStatus` function.
So current ts = math.MaxUint64 is used to make the lock outdated.

Reply:
I think violates the `currentTS` protocol is not a good idea.

Reply:
Another choice here is using the true current ts and only resolve the outdated lock.
I'm not sure the influence of the change as it changes the behavior of  GC worker. @coocood

Reply:
GC should not  resolve not expired lock.